tgui/packages/tgui/interfaces/CharacterPreferences/index.tsx

```diff
@@ -0,0 +1,183 @@
+                Appearance
+              </Tabs.Tab>
+              <Tabs.Tab onClick={() => act('open-occupation-window')}>Occupation</Tabs.Tab>
+              <Tabs.Tab onClick={() => act('open-traits-window')}>Traits</Tabs.Tab>
+              <Tabs.Tab
+                selected={menu === CharacterPreferencesTabKeys.GameSettings}
+                onClick={() => setMenu(CharacterPreferencesTabKeys.GameSettings)}>
+                Game Settings
+              </Tabs.Tab>
+              <Tabs.Tab
+                selected={menu === CharacterPreferencesTabKeys.Saves}
+                onClick={() => setMenu(CharacterPreferencesTabKeys.Saves)}>
+                Cloud Saves
+              </Tabs.Tab>
+            </Tabs>
+          </Stack.Item>
+          <Stack.Item grow="1">
+            {menu === CharacterPreferencesTabKeys.General || menu === CharacterPreferencesTabKeys.Character ? (
+              <Stack fill>
+                <Stack.Item grow="1">
+                  <Section scrollable fill>
+                    {menu === CharacterPreferencesTabKeys.General && <GeneralTab />}
+                    {menu === CharacterPreferencesTabKeys.Character && <CharacterTab />}
+                  </Section>
+                </Stack.Item>
+                <Stack.Item ml="5px">
+                  <Section fill>
+                    <ByondUi
+                      params={{
+                        id: data.preview,
+                        type: 'map',
+                      }}
+                      style={{
+                        width: '80px',
+                        height: '80px',
+                      }}
+                    />
+                    <Box textAlign="center" mt="5px">
+                      <Button icon="chevron-left" onClick={() => act('rotate-clockwise')} />
+                      <Button icon="chevron-right" onClick={() => act('rotate-counter-clockwise')} />
+                    </Box>
+                  </Section>
+                </Stack.Item>
+              </Stack>
+            ) : (
+              <Section scrollable fill>
+                {menu === CharacterPreferencesTabKeys.GameSettings && <GameSettingsTab />}
+                {menu === CharacterPreferencesTabKeys.Saves && <SavesTab />}
+              </Section>
+            )}
+          </Stack.Item>
+          <Stack.Item>
+            <Section>
+              <Button.Confirm content="Reset All" onClick={() => act('reset')} />
+            </Section>
+          </Stack.Item>
+        </Stack>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const SavesAndProfile = (_props, context) => {
+  const { act, data } = useBackend<CharacterPreferencesData>(context);
+
+  const activeProfileIndex = data.profiles.findIndex((p) => p.active);
+
+  return (
+    <Stack vertical>
+      <Stack.Item>
+        <Stack>
+          {data.profiles.map((profile, index) => (
+            <Stack.Item key={index} ml={index === 0 ? '0' : '5px'} grow="1">
```

Comment:
Unsure if `ml={index === 0 ? '0' : '5px'}` needs setting here, I *think* this might be covered by `Stack.Item`, but I could see how this could be a little weird for sure.

If it doesn't work with it off (and they all squish up), check if adding `fill` to the `Stack` two lines up solves the problem.

Reply:
Can be addressed at a later date.